### PR TITLE
Update the development setup

### PR DESCRIPTION
**Pull request by JoschkaSchulz**
I had this files unchanged from the last testing session. Maybe it would be usefull for everyone. Thanks to @salzig for most of the changes.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -89,6 +89,17 @@ This will list links and logos to all the usergroups.
...
 ### Install locally
 
+### On a Docker Container
+
+```sh
+bundle
+
+docker-compose up postgres -d
+
+./bin/rails db:setup
```
Comment (salzig) [inline readme.md:99]: Maybe we should follow rails here and use bin/setup. We could even skip the `bundle` step. 
```suggestion
./bin/setup
```

File: config/database.yml [hunk 2]
```diff
@@ -4,14 +4,15 @@ development: &default
   min_messages: WARNING
   pool: 5
   username: postgres
-  host: <%= ENV["POSTGRES_PORT_5432_TCP_ADDR"] || 'localhost' %>
+  password: <%= ENV["POSTGRES_PASSWORD"] || "UNSECURE_LOCAL_DEVELOPMENT_PASSWORD" %>
```
Comment (salzig) [inline config/database.yml:7]:
```suggestion
  password: <%= ENV["POSTGRES_PASSWORD"] || "INSECURE_LOCAL_DEVELOPMENT_PASSWORD" %>
```

File: docker-compose.yml [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
-    - .:/app
-  volumes_from:
-    - box
+services:
+  postgres:
+    image: postgres:14.12
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: UNSECURE_LOCAL_DEVELOPMENT_PASSWORD
```
Comment (salzig) [inline docker-compose.yml:7]:
```suggestion
      POSTGRES_PASSWORD: INSECURE_LOCAL_DEVELOPMENT_PASSWORD
```

File: docker-compose.yml [hunk 4]
```diff
@@ -1,16 +1,27 @@
...
+
+  box:
```
Comment (salzig) [inline docker-compose.yml:9]: If we don't suggest to use box&web anymore, should we remove them then?

Comment (JoschkaSchulz) [reply docker-compose.yml:9]: total agree

File: readme.md (hunk 1 above)
Comment (JoschkaSchulz) [reply readme.md]: yeah, I changed it and removed also bundle from the readme

File: config/database.yml (hunk 2 above)
Comment (JoschkaSchulz) [reply config/database.yml]: hehe sorry for that, is fixed

File: docker-compose.yml (hunk 3 above)
Comment (JoschkaSchulz) [reply docker-compose.yml]: hehe sorry for that, is fixed

**Pull request merged**